src/pages/WebMeetWaitingRoom.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from "react";
+import "../App.css";
+import "./home.css";
+import * as patientService from "../services/patient";
+import { getDateString, getTimeString, getToday } from "../utils";
+import { BOOKING_STATUS } from "../constants/constants";
+
+const WebMeetWaitingRoom = ({ patientDetails }) => {
+  const [appointment, setAppointment] = useState({});
+  const today = getToday();
+  useEffect(() => {
+    window.scrollTo(0, 0);
+    getAppointmentDetail();
+  }, []);
+
+  const getAppointmentDetail = async () => {
+    try {
+      const response = await patientService.getAppointmentDetailsByPatientId(
+        patientDetails.patientId
+      );
+      setAppointment(response.appointment);
+
+      response.appointment.eventStatusDesc === BOOKING_STATUS.confirmed &&
```

Comment:
let's make sure `response.appointment` is not undefined before using it. We can either use `&&` or `?.` operators

Reply:
resolved